docs/release-notes/known-issues.md

```diff
@@ -29,4 +29,29 @@ On 04/16/2025, a known issue was discovered that can cause a crash of the Event
 As a workaround, and to prevent crashes, Beats will ignore any filters provided when working with forwarded events on Windows Server 2025 until the issue is resolved.
 
 % **Resolved**
-% On [Month/Day/Year], this issue was resolved.
\ No newline at end of file
+% On [Month/Day/Year], this issue was resolved.
+
+:::
+
+:::{dropdown} Filebeat's Filestream input does not validate `clean_inactive`.
+**Applies to**: Filebeat < 9.2.0
+
+The Filestream input does not enforce the restrictions documented for
+the `clean_inactive` option, thus allowing configurations that can
+lead to data re-ingestion issues.
+
+**Fixed in**: 9.2.0
+:::
+
+:::{dropdown} Setting clean_inactive: 0 in Filebeat's Filestream input will cause data to be re-ingested on every restart.
+**Applies to**: Filebeat >= 8.14.0 and < 9.2.0
+
+When `clean_inactive: 0` Filestream will clean the state of all files
+on start up, effectively re-ingesting all files on restart.
```

Comment:
Is there an issue/PR you can link to here, too?

Reply:
The same thing as https://github.com/elastic/beats/pull/46392#discussion_r2324853790